src/app/features/courses/courses.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Input } from '@angular/core';
+import { CourseDataType } from '@app/shared/types/course.model';
+
+@Component({
+  selector: 'app-courses',
+  templateUrl: './courses.component.html',
+  styleUrls: ['./courses.component.scss']
+})
+export class CoursesComponent {
+  @Input() courses:CourseDataType[]=[];
+  showCourseInfo:CourseDataType | undefined=undefined;
+
+  readonly emptyListTitle = 'Your List is Empty';
+  readonly emptyListText = "Please use 'ADD NEW COURSE' button to add your first course";
+
+  handleShowCourse(courseId:CourseDataType['id']) {
+    this.showCourseInfo = this.courses.find((course)=>course.id===courseId);
+  }
+
+  handleShowCourses() {
```

Comment:
Following the above approach: `this.showCourseInfo = !this.showCourseInfo;`

Reply:
see comment : https://github.com/momanten/angular-fundamentals-template/pull/1#discussion_r1778148079

Reply:
ok, noted